src/gui/search/searchsortmodel.h

```diff
@@ -49,8 +51,44 @@ class SearchSortModel: public QSortFilterProxyModel
 
     explicit SearchSortModel(QObject *parent = 0);
 
+    void enableNameFilter(bool enabled);
+    void setNameFilter(const QString& searchTerm = QString());
+
+    //! \brief setSizeFilter Sets parameters for filtering by size
+    //! \param minSize minimal size in bytes
+    //! \param maxSize maximal size in bytes, negative value to disable filtering
+    void setSizeFilter(qint64 minSize, qint64 maxSize);
+
+     //! \brief setSeedsFilter Sets parameters for filtering by seeds number
+     //! \param minSeeds minimal number of seeders
+     //! \param maxSeeds maximal number of seeders, negative value to disable filtering
+    void setSeedsFilter(int minSeeds, int maxSeeds);
+
+     //! \brief setLeechsFilter Sets parameters for filtering by leechs number
+     //! \param minLeechs minimal number of leechers
+     //! \param maxLeechs maximal number of leechers, negative value to disable filtering
+    void setLeechsFilter(int minLeechs, int maxLeechs);
+
+    bool isNameFilterEnabled() const;
+
+    QString searchTerm() const;
+
+    int minSeeds() const;
+    int maxSeeds() const;
+
+    qint64 minSize() const;
+    qint64 maxSize() const;
+
 protected:
     virtual bool lessThan(const QModelIndex &left, const QModelIndex &right) const;
+    virtual bool filterAcceptsRow(int source_row, const QModelIndex &source_parent) const;
```

Comment:
Yes. I already wrote this: you already corrected it. GitHub just shows the old comment until now (probably because it is linked to the previous line).